### PR TITLE
copy() and text fixes

### PR DESCRIPTION
**Pull request by jefarrell**
Switching .get() to .copy()
Updated text to new p5 methods

### CODE REVIEW

File: chp02_forces/NOC_2_05_fluidresistance/sketch.js [hunk 1]
```diff
@@ -29,7 +27,9 @@ function setup() {
 }
 
 function draw() {
-  background(127);
+  background(255);
+  fill(0);
+  text("click to reset",10,100);
```
Comment (shiffman) [inline chp02_forces/NOC_2_05_fluidresistance/sketch.js]: Hi @jefarrell, I was actually using p5.dom for instructions to appear below canvas as `<p>` elements.  Was that not working for you?

Comment (jefarrell) [inline chp02_forces/NOC_2_05_fluidresistance/sketch.js]: Ah ok, I totally missed that.  Ignore those text changes then - the p5.dom works fine.  Sorry about that!

Comment (shiffman) [inline chp02_forces/NOC_2_05_fluidresistance/sketch.js]: No worries, you'll have to add new commits that revert if you want the fame and glory that comes with having your pull request merged!

Comment (jefarrell) [inline chp02_forces/NOC_2_05_fluidresistance/sketch.js]: I added a new commit with reverted text elements.  All the glory!

**Comment by shiffman** [comment]
Thank you!

**Pull request merged**